### ISSUE TITLE
Bloody Zombies (appid 449640)

### ISSUE THREAD

**Issue by Patola** [ValveSoftware/Proton#5189]
# Compatibility Report
- Name of the game with compatibility issues: Bloody Zombies
- Steam AppID of the game: 449640

## System Information
- GPU: RX 6800 XT
- Driver/LLVM version: 21.2.3
- Kernel version: 5.14.8-203-tkg-bmq
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Patola/a51205dcfa0fb19e3acbd57217263b72
- Proton version: 6.3-7

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 

Game doesn't even start, be it in pancake mode or VR mode.

## Reproduction

Try and start the game. Doesn't start.

Added the logs attempting both VR and pancake modes (with SteamVR off).

[steam-449640.log](https://github.com/ValveSoftware/Proton/files/7267015/steam-449640.log)
[steam-449640-vr.log](https://github.com/ValveSoftware/Proton/files/7267017/steam-449640-vr.log)

**Comment by deadmeme5** [comment]
do you have proper Read Write Exec access on the specified disk? Are you using NTFS, NTFS-3G or EXT4??

**Comment by Patola** [comment]
of course. I am using ext4. I have hundreds of installed games working in the same prefix: native games, proton games, proton games with VR.

**Comment by deadmeme5** [comment]
> of course. I am using ext4. I have hundreds of installed games working in the same prefix: native games, proton games, proton games with VR.

tried using NTFS?

**Comment by Patola** [comment]
> > of course. I am using ext4. I have hundreds of installed games working in the same prefix: native games, proton games, proton games with VR.
> 
> tried using NTFS?

No, but sorry, why would I do this? NTFS is worse for making games work than ext4. I use LVM and I could easily do that by creating a new logical volume formatted with NTFS, but I see no chance of it working better than ext4.

**Comment by deadmeme5** [comment]
> No, but sorry, why would I do this? NTFS is worse for making games work than ext4. I use LVM and I could easily do that by creating a new logical volume formatted with NTFS, but I see no chance of it working better than ext4.

I mean your problem log is wine not being able to access the game no? NTFS-3G solved this for me, at least try setting the disk as you own it with a uid on /cat/passwd | grep USERNAMEHERE and see if that works.

**Comment by Patola** [comment]
> I mean your problem log is wine not being able to access the game no? NTFS-3G solved this for me, at least try setting the disk as you own it with a uid on /cat/passwd | grep USERNAMEHERE and see if that works.

Interesting consideration. You might as well be right. Since it fails so early, it's possible it can't find the executable or some component/dynamic library (although I don't see a message pointing to this). I've seen this happen because of capitalization, since Windows is not case-sensitive (it's "case retentive but not case sensitive") and Linux and its filesystems are. So, if NTFS sometimes solves that for you might be due to that. This has been recently changed in wine, not sure it will propagate to Proton.

Unfortunately, to be frank, this was one game I bought just for curiosity, it's not much my style. Seeing that it did not work, I refunded it. So I can't test it now,

**Comment by Patola** [comment]
Seems this game returned to my library, I don't even recall how I got it back, maybe in a bundle? Still not starting under Proton 8.0-5.
[steam-449640.log](https://github.com/ValveSoftware/Proton/files/14786633/steam-449640.log)
